spec/controllers/hyrax/generic_works_controller_spec.rb

```diff
@@ -486,14 +486,19 @@
   end
 
   describe '#manifest' do
-    let(:work) { create(:work_with_one_file, user: user) }
-    let(:file_set) { work.ordered_members.to_a.first }
     let(:manifest_factory) { double(to_h: { test: 'manifest' }) }
 
+    let(:file_set) do
+      create_for_repository(:file_set,
+                            user: user,
+                            content: file)
+    end
+    let(:file) { fixture_file_upload('/world.png', 'image/png') }
+    let(:work) do
+      create_for_repository(:work, user: user, member_ids: [file_set.id])
```

Comment:
Does this need to be a public work in order for the "produces a manifest" test to pass? I'm getting an Authorization Required result.

Reply:
I don't know, I just changed it such that it wasn't trying to attach files using the Hydra::Works services.